src/components/AppButton/AppButton.test.tsx

```diff
@@ -0,0 +1,32 @@
+import { render, screen } from '@testing-library/react';
+
+import { AppButton } from './AppButton';
+
+describe('Test AppButton component', () => {
+  test('Should render Google Play Button correctly', () => {
+    render(
+      <AppButton variant="appButton" link="">
+        Google Play
+      </AppButton>,
+    );
+
+    expect(screen.getByText(/Google Play/i)).toBeInTheDocument();
+    expect(screen.getByText(/Google Play/i)).toHaveClass('text-sm');
+    expect(screen.getByText(/Google Play/i)).toHaveClass('font-bold');
+    expect(screen.getByRole('button')).toBeInTheDocument();
+  });
+
+  test('Should render App Store Button correctly', () => {
```

Comment:
The second test is extra step. I will give a article that will explain it. Sometimes you only need to check less tests because they do not add any new value. These 2 are identical, the difference is only the data you pass, but the logic is the same. So one of them can be removed